packages/get-starknet/src/index.ts

```diff
@@ -20,6 +20,7 @@ const defaultStarknetWindowObject: StarknetWindowObject = {
     }
     return fail()
   },
+  disconnect: async () => {},
```

Comment:
Should be like others:

```suggestion
  disconnect: fail,
```